src/EnergyPlus/HVACDXHeatPumpSystem.cc

```diff
@@ -468,36 +466,33 @@ namespace HVACDXHeatPumpSystem {
         using EMSManager::CheckIfNodeSetPointManagedByEMS;
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        int ControlNode; // control node number
-        int DXSysIndex;
-        int OutdoorAirUnitNum;    // "ONLY" for ZoneHVAC:OutdoorAirUnit
-        Real64 OAUCoilOutletTemp; // "ONLY" for zoneHVAC:OutdoorAirUnit
```

Comment:
[src/EnergyPlus/HVACDXHeatPumpSystem.cc:472]:(style),[variableScope],The scope of the variable 'DXSysIndex' can be reduced.
[src/EnergyPlus/HVACDXHeatPumpSystem.cc:473]:(style),[variableScope],The scope of the variable 'OutdoorAirUnitNum' can be reduced.